src/plugins/data/server/search/opensearch_search/opensearch_search_strategy.ts

```diff
@@ -63,13 +63,17 @@ export const opensearchSearchStrategyProvider = (
 
       // ignoreThrottled is not supported in OSS
       const { ignoreThrottled, ...defaultParams } = await getDefaultSearchParams(uiSettingsClient);
-
+      console.log("these are the default params",defaultParams)
       const params = toSnakeCase({
         ...defaultParams,
         ...getShardTimeout(config),
         ...request.params,
       });
-
+      if (params.body.pit) {
```

Comment:
We will need to make changes in the search strategy as well since there are some fields which can't be passed in PIT search object.